### ISSUE TITLE
question 

### ISSUE THREAD

**Issue by mike18400** [xuzhengyi1995/Manga_downloader#104]
for the new version do i still need to download selenium and the other stuff ?

**Comment by xuzhengyi1995** [comment]
No, for the new Chromium browser method, only need to download the pack and use it. No need to download seleniumn or Python.
